csharp-basics/exercises/Polymorphism/.gitignore

```diff
@@ -0,0 +1,7 @@
+﻿
+# Ignorē kompilācijas iznākumu direktorijas
+bin/
+obj/
+
```

Comment:
ir tikai 1 gitignore, citi vairs nav jāliekt, Un pat ja liktu to šeit, tas neko nedarītu. Gitignore ir jābūt pirms taisa branchu

Reply:
tagad es to sapratu, ka jāliek pirms brancha, bet visus gitignore taisīju tieši pēc brancha izvedidošanas